libnopegl/src/path.c

```diff
@@ -142,7 +160,14 @@ int ngli_path_bezier3_to(struct path *s, const float *ctl0, const float *ctl1, c
 
 int ngli_path_close(struct path *s)
 {
-    return ngli_path_line_to(s, s->origin);
+    int ret = ngli_path_line_to(s, s->origin);
```

Comment:
What happens if:
- the last segment has already the closing flag ? 
- ngli_path_close() is called multiple time ?

Reply:
I added a commit "path: prevent no-ops when adding lines or bézier curves" which should address these concerns.